src/main/java/teammates/ui/controller/FeedbackSubmissionEditSaveAction.java

```diff
@@ -162,6 +163,22 @@ protected ActionResult execute() throws EntityDoesNotExistException {
                 errors.add(String.format(Const.StatusMessages.FEEDBACK_RESPONSE_INVALID_RECIPIENT, questionIndx));
             }
 
+            // If there are duplicate values in recipient list (except empty strings),
+            // trigger this error.
+            for (String recipient : responsesRecipients) {
```

Comment:
Is there a reason why you check for duplicates here instead of in the previous loop where `responseRecipient` is populated (line 135)?

Reply:
Are you proposing that I should throw the error while adding the element (I mean first checking if the same recipient is present in the `responsesRecipients` list, if it exists, throwing this error, otherwise adding the recipient in the list?)

Reply:
Yeah that's what I meant. That way we can avoid iterating the list twice?

Reply:
Okay. Good idea. Will update in the next push.